jira/client.py

```diff
@@ -176,8 +176,9 @@ def __iter__(self) -> Iterator[ResourceType]:
         return super().__iter__()
 
     # fmt: off
-    # The mypy error we ignore is about returning a contravariant type. As this class is a List of a generic 'Resource' class
-    # this is the right way to specify that the output is the same as which the class was initialized with.
+    # The mypy error we ignore is about returning a contravariant type. As this class
+    # is a List of a generic 'Resource' class this is the right way to specify that the
+    # output is the same as which the class was initialized with.
```

Comment:
```suggestion
    # The mypy error we ignore is about returning a contravariant type. 
    # As this class is a List of a generic 'Resource' class
    # this is the right way to specify that the output is the same
    # as which the class was initialized with.
```

Reply:
lol isnt this nit picking alot :-D

I can also revert all changes in inline comments as i think all other changes are without remarks